index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Habby's Page</title>
+	<link rel="stylesheet" type="text/css" href="style.css" >
+	<meta charset="UTF-8">
+
+</head>
+<body>
+
+<img src="http://i.imgur.com/enRYXYj.jpg?1" alt="HabbyPic" title="HabbyPic">
```

Comment:
I like that you included an `alt` and a `title` attribute, but in both cases, they should be a more complete description of the picture, e.g. "A picture of me, Habby".